pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,322 @@ excerpt:
+
+# Requirements
+
+- Advanced installation method
+In order to install DC/OS Enterprise with a custom CA certificate you have to use the [advanced DC/OS installation method](/1.10/installing/custom/advanced/). Other installation methods are not supported.
+
+- Custom CA certificate
+- Private key associated with the custom CA certificate
+- Certificate chain associated with the custom CA certificate
+
+The files containing the custom CA certificate, the associated private key and the certificate chain has to be placed in the predefined locations where the installer can find them during the installation. (see the next sections for details)
+
+
+## Manually placing the the custom CA certificate, the associated private key and the certificate chain onto the bootstrap node
+The custom CA certificate, the associated private key and the certificate chain files have to be put in the `genconf/` directory on the bootstrap node. The paths to these files have to be specified in the configuration file.
+
+## Manually placing the private key associated with the custom CA certificate onto the master nodes
+
+For security reasons, the installer will not copy the private key from the bootstrap node to the master nodes. 
+The private key associated with the custom CA certificate has to be distributed manually to every DC/OS master node **before starting the installation**. The filesystem path for the private key file has to be `/var/lib/dcos/pki/tls/CA/private/custom_ca.key`. The directory `/var/lib/dcos/pki/tls/CA/private` should be created manually with the following  command before putting the file `custom_ca.key` in the directory on every DC/OS master node:
+
+```bash
+ mkdir -p /var/lib/dcos/pki/tls/CA/private
+```
+
+Furthermore, the file has to be owned by the root Unix user and to have 0600 permissions set.
+
+If you copy private key file over the network onto the master nodes, the network channel has to be adequately protected.
+
+An example of copying the CA private key is given below. The commands are executed on the bootstrap node. The `W.X.Y.Z` denotes the IP address of a master node below.
+
+```bash
+cd $DCOS_INSTALL_DIR/config
+scp custom_ca.key centos@W.X.Y.Z:/var/lib/dcos/pki/tls/CA/private
+```
+
+## Specifying the locations of the custom CA certificate, the associated private key and the certificate chain files in the configuration file
+
+The filesystem paths to the custom CA certificate, associated private key and certificate chain files in the `genconf` directory on the bootstrap node have to be specified in the configuration file using, respectively, the `ca_certificate_path`, `ca_certificate_key_path` and  `ca_certificate_chain_path` parameters.
+The paths to the custom certificate, the associated private key and the certificate chain must be relative paths and must not contain the `genconf` prefix.
+
+[This section](#example-use-cases) shows how to set these configuration parameters.
```

Comment:
The current link looks like it might link to a separate page. Indicate to the reader that they can keep reading linearly, and will get to this information:

The [Example Use Cases] (#example-use-cases) section below shows...